docker-compose.yml

```diff
@@ -86,6 +86,13 @@ services:
     environment:
       - POSTGRES_HOST=postgres_docker
       - NODE_ENV=development
+  sftp:
+    image: jmcombs/sftp
+    volumes:
+        - ./test-sftp:/home/tta_ro/ProdTTAHome
+    ports:
+        - "${SFTP_EXPOSED_PORT:-22}:22"
```

Comment:
Let's add `SFTP_EXPOSED_PORT` to `.env.example` with a default of something other than 22 (2222?)